avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -337,17 +336,17 @@ mucoulomb = 0.155
 mu0wetsnow = 0.2
 # xsiWetSnow
 xsiwetsnow = 981.
-# reference enthalpy in J/kg to compute the new mu = mu_0 x exp(-enthalpy/enthRef)
+# reference enthalpy to compute the new mu = mu_0 x exp(-enthalpy/enthRef) [J/kg]
 enthRef = 20000
-# initial temperature in °C
+# initial temperature [°C]
 TIni = -10
-# initial temperature for entrainment in °C
+# initial temperature for entrainment [°C]
 entTempRef = -10.0
 # J/kgK (ice)
 cpIce = 2050.0
```

Comment:
I believe this parameter is not explained in the documentation https://docs.avaframe.org/en/1.8.3/theoryCom1DFA.html#wet-snow-friction-type. Is this maybe a recent addition?

Reply:
cpIce

Reply:
It is mentioned in the text, I.e the specific heat capacity.
I guess a good idea would be to add the formular for the totalEnthalpy to the documentation right away. See com1DFA.py ll. 1247 for the relevant code. If you want to try, give it a go. The relevant documentation source is theoryCom1DFA.rst